index.html

```diff
@@ -1403,24 +1403,17 @@
   <div class="head">
    <p data-fill-with="logo"></p>
    <h1 class="p-name no-ref" id="title">AV1 Codec ISO Media File Format Binding</h1>
-   <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="content">Living Standard, <time class="dt-updated" datetime="2018-08-16">16 August 2018</time></span></h2>
+   <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="content">v1.0.0, <time class="dt-updated" datetime="2018-08-31">31 August 2018</time></span></h2>
```

Comment:
Did you manually edit the HTML to produce this? (just wondering)

Reply:
Yes. A bit ugly but I don't know how to do it otherwise. Anyway, when we update the spec again we should restore the "Warning" part and that sub-title could become "Editor's draft" or alike to make it clear that this is not the approved spec.

Reply:
Yeah, unfortunate, but I don't know what else we can do with bikeshed at present. Thanks!